docs/modules/literate-programming.md

```diff
@@ -0,0 +1,16 @@
+# Prelude Literate Programming
+
+Prelude's `prelude-literate-prgg` module enables some additional functionality
```

Comment:
Why not `prelude-literate-programming` instead? `prgg` is a bit hard to parse by humans IMO. :-)